test/linter/test-obsolete.ts

```diff
@@ -45,10 +51,9 @@ export const neverImplemented = (support: InternalSupportBlock): boolean => {
   return true;
 };
 
-const errorTime = new Date(),
-  warningTime = new Date();
+export const errorTime = new Date();
 errorTime.setFullYear(errorTime.getFullYear() - 2.5);
```

Comment:
This isn't new, but it is broken. The argument is an integer, so the 2.5 is equivalent to 3, as shown by this test in a node CLI:
```
> errorTime = new Date();
2023-05-23T13:08:50.436Z
> errorTime.setFullYear(2023 - 2.5)
1590239330436
> errorTime.setFullYear(2023 - 3)
1590239330436
```

So, effectively `errorTime` is 3 years in the past, and `warningTime` is 2 years in the past. Because `errorTime` is always less than `warningTime`, `errorTime` is effectively unused, the second condition here is unreachable:

https://github.com/mdn/browser-compat-data/blob/a3c9e32d2e169c41cb5260f08dd1d02fd460e742/test/linter/test-obsolete.ts#L101-L108

Can you drop `errorTime` and then maybe just use `twoYearsAgo` directly instead of `warningTime`?

Reply:
Three years seems alright, but let's make those sorts of changes in another PR instead of in this one?